client/less/classic-modal.less

```diff
@@ -0,0 +1,45 @@
+.success {
```

Comment:
It seems like you are replicating the css here. This should already exist in our codebase.